config/sidekiq_scheduler.yml

```diff
@@ -165,3 +165,8 @@ VaForms::FetchLatest:
   cron: "0 0 * * * America/New_York"
   class: VaForms::FormReloader
   description: "Fetches latest VA Forms"
+
+AppsApi::ReloadAppsList:
+  cron: "0 */8 * * * America/New_York"
+  class: AppsApi::DirectoryReloader
```

Comment:
I don't see any changes for the AppsApi::DirectoryReloader class in this PR

Reply:
This was a piece of dead code that slipped through. It's been removed